docs/details/develop-checks/checks-v2.md

```diff
@@ -0,0 +1,223 @@
+    | MONGODB_GETCMDLINEOPTS          |    Executes db.adminCommand( { getCmdLineOpts: 1 } ) against MongoDB's "admin" database. For more information, see [getCmdLineOpts](https://docs.mongodb.com/manual/reference/command/getCmdLineOpts/) |No|
+    | MONGODB_REPLSETGETSTATUS     |   Executes db.adminCommand( { replSetGetStatus: 1 } ) against MongoDB's "admin" database. For more information, see  [replSetGetStatus](https://docs.mongodb.com/manual/reference/command/replSetGetStatus/) |No|
+    | MONGODB_GETDIAGNOSTICDATA |Executes db.adminCommand( { getDiagnosticData: 1 } ) against MongoDB's "admin" database. For more information, see [MongoDB Performance](https://docs.mongodb.com/manual/administration/analyzing-mongodb-performance/#full-time-diagnostic-data-capture)| No|
+    | METRICS_INSTANT |Executes instant [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html) query. Query can use placeholders in query string {% raw %} **{{.NodeName**}} and **{{}}.ServiceName}}**  {% endraw %}. Both match target service/node names. To read more about instant queries see [Prometheus docs](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries).|Yes|
+    | METRICS_RANGE |Executes range [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html) query. Query can use placeholders in query string {% raw %} **{{.NodeName**}} and **{{}}.ServiceName}}**  {% endraw %}. Both match target service/node names. To read more about range queries see [Prometheus docs](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries).|Yes|
+
+## Query parameters
+- **METRICS_INSTANT**
+  - **lookback** (duration, optional): specifies how far in past to look back to metrics history. If this parameter is not specified, then query executed on the latest data. Example values: 30s, 5m, 8h.
+- **METRICS_RANGE**
+  - **lookback** (duration, optional): specifies how far in past to look back to metrics history. If this parameter is not specified, then query executed on the latest data. Example values: 30s, 5m, 8h.
+  - **range** (duration, required): specifies time window of the query. This parameter is equal to [prometheus API](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries).
+  - **step** (duration, required): query resolution. This parameter is equal to [prometheus API](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries).
+
+
+## Develop version 2 checks
+To develop custom checks for PMM 2.28 and later: 
+
+1. Install the latest PMM Server and PMM Client builds following the [installation instructions](https://www.percona.com/software/pmm/quickstart#). 
+2. Run PMM Server with special environment variables:
+ 
+    - _PMM_DEBUG=1_ to enable debug output that would be useful later;
+    - _PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml_ to use checks from the local files instead of downloading them from Percona Portal.
```

Comment:
```suggestion
    - _PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml_ to use checks from the local files instead of downloading them from Percona Platform.
```